cilium/cmd/status.go

```diff
@@ -95,7 +95,17 @@ func statusDaemon() {
 			os.Exit(1)
 		}
 
-		healthPkg.GetAndFormatHealthStatus(w, true, allHealth, healthLines)
+		healthEnabled := false
+		for _, c := range sr.Controllers {
+			if c.Name == "cilium-health-ep" {
```

Comment:
nit: this relies on the name of the controller being named exactly `cilium-health-ep` and this code will silently break if the name is changed. Maybe extract `cilium-health-ep` as a constant and use it in both places (other is [here](https://github.com/cilium/cilium/blob/master/daemon/cmd/health.go#L53)) to make the implicit coupling explicit?